content/en/docs/concepts/services-networking/connect-applications-service.md

```diff
@@ -338,7 +338,7 @@ NAME                               READY     STATUS    RESTARTS   AGE
 curl-deployment-1515033274-1410r   1/1       Running   0          1m
 ```
 ```shell
-kubectl exec curl-deployment-1515033274-1410r -- curl https://my-nginx --cacert /etc/nginx/ssl/nginx.crt
+kubectl exec curl-deployment-1515033274-1410r -- curl -k https://my-nginx --cacert /etc/nginx/ssl/nginx.crt
```

Comment:
This looks like the wrong fix. Why doesn't `/etc/nginx/ssl/nginx.crt` contain a CA that `curl` considers valid?

Reply:
How about migrating this to a Task and fixing it for readability at the same time? (see #15560 
I can help out with advice.